### PR TITLE
Multiply

### PR DESCRIPTION
**Pull request by manvithalva12**

### CODE REVIEW

File: multiply.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+function multiply(a,b) {
```
Comment (Rajharsh2) [inline multiply.js]: - provide json documentaion for better understanding.

**Review by Rajharsh2 — CHANGES_REQUESTED** [review]

File: multiply.js (hunk 1 above)
Comment (albin1616) [inline multiply.js]: change function name

**Review by albin1616 — CHANGES_REQUESTED** [review]

File: multiply.js (hunk 1 above)
Comment (manvithalva12) [reply multiply.js]: resolved

Comment (Rajharsh2) [reply multiply.js]: Look good to me

**Review by Rajharsh2 — APPROVED** [review]

File: multiply.js (hunk 1 above)
Comment (manvithalva12) [reply multiply.js]: resolved

Comment (albin1616) [reply multiply.js]: looking good to me

**Review by albin1616 — APPROVED** [review]

**Pull request merged**